src/app/rosetta/lib/construction.ml

```diff
@@ -158,11 +158,18 @@ module Metadata_data = struct
   type t =
     { sender: string
     ; nonce: Unsigned_extended.UInt32.t
-    ; token_id: Unsigned_extended.UInt64.t }
+    ; token_id: Unsigned_extended.UInt64.t
+    ; minimum_fee: Amount.t list [@default []] }
```

Comment:
Seems a little strange that this is a list. Can it not be an `option` (`default None`) instead?

Reply:
can try... 👍🏼

Reply:
Done!